__tests__/app.test.js

```diff
@@ -85,4 +86,26 @@ describe("nc-news", () => {
         });
     });
   });
+
+  describe("GET api/articles", () => {
+    test("GET 200: returns with all articles sorted via date in descending order when no query given", () => {
+      return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({ body }) => {
+          const { articles } = body;
+          expect(articles.length).toBe(13);
+          articles.forEach((article) => {
+            expect(typeof article.author).toBe("string");
+            expect(typeof article.title).toBe("string");
+            expect(typeof article.article_id).toBe("number");
+            expect(typeof article.topic).toBe("string");
+            expect(typeof article.created_at).toBe("string");
+            expect(typeof article.votes).toBe("number");
+            expect(typeof article.comment_count).toBe("number");
+          });
+          expect(articles).toBeSortedBy("created_at", { descending: true });
+        });
```

Comment:
Good use of jest sorted

You may want to consider using toMatchObject here, just to make your test a little cleaner